frontend/src/components/VideoQualityLabel/VideoQualityLabel.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import type { Meta, StoryObj } from '@storybook/react-vite';
+
+import VideoQualityLabel from './VideoQualityLabel';
+
+const meta: Meta<typeof VideoQualityLabel> = {
+  title: 'Components/VideoQualityLabel',
+  component: VideoQualityLabel,
+  tags: ['autodocs'],
+};
+
+export default meta;
+type Story = StoryObj<typeof VideoQualityLabel>;
+
```

Comment:
The first story should one called `Default` this is gonna be the first to show in the documentation. So the Default should be something like:

```suggestion
export const Default: Story = {
  args: {
    type: 'HD',
  },
};
```